controller/client/client.go

```diff
@@ -31,6 +32,89 @@ type Client struct {
 	*httpclient.Client
 }
 
+type JobWatcher struct {
+	events chan *ct.JobEvent
+	stream stream.Stream
+}
```

Comment:
Then I need to move all the JobWatcher receiving methods to the types package as well. From what I could tell, that package was avoiding having types that receive methods. Did I misunderstand? Is there a cleaner way to do this?